src/components/Item/VisualizationItem/Item.js

```diff
@@ -162,11 +162,11 @@ export class Item extends Component {
             case REPORT_TABLE: {
                 return (
                     <>
-                        {!this.state.pluginIsLoaded ? (
-                            <div className={classes.loadingCover}>
+                        {!this.state.pluginIsLoaded && (
+                            <div style={props.style} className={classes.cover}>
```

Comment:
It seems like the `react-grid-item` already has a static height so this could be solved with `display:flex` at the top, `flex-grow: 1` on the `dashboard-item-content` and `height:100%` on further nested children ![image](https://user-images.githubusercontent.com/12590483/80185827-4e070080-860d-11ea-8ba0-570b77cf7aab.png)

Reply:
To do this, I will need to add a div that wraps the Header and dashboard-item-content. Since it isn't possible to add any css to react-grid-item (it is generated by the React Grid Layout library)

Reply:
Actually, we can add it old-school, by targeting the react-grid-item class directly.

Reply:
On further investigation, I found a commit that changed how we did this height calculation. Turns out we previously used display: flex as you are recommending, but had to change it in this commit: https://github.com/dhis2/dashboards-app/commit/fab4d387fc9c60528301570e49bf4042b662e625). That was two years ago, and I think it is worth revisiting, but in a separate commit/PR.